### PR TITLE
Html2parquet example

### PR DESCRIPTION
**Pull request by touma-I**
## Why are these changes needed?

This PR includes a example showing how html2parquet can be used in a notebook 

## Related issue number (if any).

https://github.com/IBM/data-prep-kit/issues/788

### CODE REVIEW

**Comment by touma-I** [comment]
@shahrokhDaijavad @sungeunan-ibm Before I forget and drop this thread, please review/edit/approve as you see fit. this is the notebook showing how to invoke the html2parquet from within a notebook that I created last week. It can be further evolved based on direct reviews. Thanks

File: transforms/language/html2parquet/notebooks/html2parquet.ipynb [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+   "execution_count": 3,
+   "id": "e75f6922-eb0f-4164-a536-f96393e04604",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ast\n",
+    "\n",
+    "# create parameters\n",
+    "local_conf = {\n",
+    "    \"input_folder\": \"input\",\n",
```
Comment (sungeunan-ibm) [inline transforms/language/html2parquet/notebooks/html2parquet.ipynb:40]: /path/to/your/input/folder
/path/to/your/output/folder

^to make sure it's the folder not files.

File: transforms/language/html2parquet/notebooks/html2parquet.ipynb [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    "    \"output_folder\": \"output\",\n",
+    "}\n",
+    "\n",
+    "params = {\n",
+    "    # Data access. Only required parameters are specified\n",
+    "    \"data_local_config\": ParamsUtils.convert_to_ast(local_conf),\n",
+    "    \"data_files_to_use\": ast.literal_eval(\"['.html']\"),\n",
```
Comment (sungeunan-ibm) [inline transforms/language/html2parquet/notebooks/html2parquet.ipynb:47]: "data_files_to_use": ast.literal_eval("['.zip', '.html']"),

support zip as well

File: transforms/language/html2parquet/notebooks/html2parquet.ipynb [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pyarrow.parquet as pq\n",
+    "import pandas as pd\n",
+    "table = pq.read_table('output/ai-alliance-index.parquet')\n",
```
Comment (sungeunan-ibm) [inline transforms/language/html2parquet/notebooks/html2parquet.ipynb:157]: table = pq.read_table('/path/to/your/output/folder/sample.parquet')

users don't have "ai-alliance-index.parquet" so making it general.

**Review by sungeunan-ibm — CHANGES_REQUESTED** [review]
I left some comments.

**Comment by touma-I** [comment]
Superseded by another PR

**Pull request closed without merging**